speed-dreams/source-2.2.3/src/drivers/assistedhuman/Driver.cpp

```diff
@@ -38,7 +41,18 @@ void Driver::InitTrack(tTrack* p_track, void* p_carHandle, void** p_carParmHandl
     m_humanDriver.init_track(m_index, p_track, p_carHandle, p_carParmHandle, p_situation);
     m_humanDriver.SetRecorder(useRecorder ? m_recorder : nullptr);
 
+    auto start = std::chrono::system_clock::now();
```

Comment:
Why is this done in the driver and not in the mediator? Right now this means only the assisted driver tests this but for example the replay driver doesn't

Reply:
cuz I need to log which is kinda frontend and if you want to display a message later you would have to move it again anyway